edenia/eden-spend-explorer
# feat(webapp): UI for smart contract integration

### UI for smart contract integration 

### What does this PR do?

- Resolve #178
- Resolve #18 
- Resolve #19 
- Resolve #24 
- Resolve #23
- Resolve #21 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. click on spend tools and see the new features

#### CheckList
